deployment/7_deploy_2.5.js

```diff
@@ -133,35 +133,49 @@ const main = async () => {
   const AugustusSwapperWrapper = await IAugustusSwapper.at(config.defi.paraswap.contract);
   await DappRegistryWrapper.addDapp(0, await AugustusSwapperWrapper.getTokenTransferProxy(), OnlyApproveFilterWrapper.address);
 
-  // AaveV2
-  console.log("Deploying AaveV2Filter");
-  const AaveV2FilterWrapper = await AaveV2Filter.new();
-  console.log(`Deployed AaveV2Filter at ${AaveV2FilterWrapper.address}`);
-  await DappRegistryWrapper.addDapp(0, config.defi.aave.contract, AaveV2FilterWrapper.address);
-
-  // Balancer
-  console.log("Deploying BalancerFilter");
-  const BalancerFilterWrapper = await BalancerFilter.new();
-  console.log(`Deployed BalancerFilter at ${BalancerFilterWrapper.address}`);
-  for (const pool of (config.defi.balancer.pools)) {
-    console.log(`Adding filter for Balancer pool ${pool}`);
-    await DappRegistryWrapper.addDapp(0, pool, BalancerFilterWrapper.address);
-  }
+  // The following filters can't be setup on Ropsten due to tha lack of integrations
```

Comment:
I believe we should also add the `refundAddress` to the DappRegistry with no filter.

Reply:
My mistake, already added.